explainability/explainability-service-rest/src/main/java/org/kie/kogito/explainability/rest/ExplainabilityApiV1.java

```
@@ -64,6 +72,16 @@ public ExplainabilityApiV1(
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     public Uni<Response> explain(ExplainabilityRequestDto requestDto) {
+        Set<ConstraintViolation<ExplainabilityRequestDto>> violations = validator.validate(requestDto);
```

As far I know, the annotation @Valid works fine with reactive endpoints, so if we annotate the requestDto field with @Valid, it should raise 400 Invalid Request directly.

---

Thanks @Sgitario , this module was new for me, good to know :) lmkwyt